pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.black]
 line-length = 99
-target-version = ['py27', 'py35', 'py36', 'py37', 'py38', 'py39', 'py310']
+target-version = ['py27', 'py35', 'py36', 'py37', 'py38']
```

Comment:
This should still test Python 3.9/3.10

Reply:
Sorry, that was me experimenting with my black installation. Reverted now